common/lib/ktransfer.ts

```diff
@@ -25,15 +25,22 @@ const transferNftResponse = async (
 
   const { contract, supports } = await collectionGetContract(chainId, address, owner.provider);
```

Comment:
should pass owner as last parameter, then no need of connectedContract afterwards